clash-protocols.cabal

```diff
@@ -148,6 +149,7 @@ library
     Protocols.Axi4.WriteAddress
     Protocols.Axi4.WriteData
     Protocols.Axi4.WriteResponse
+    Protocols.Axi4.Stream.Axi4Stream
```

Comment:
How come this is called `Stream.Axi4Stream`? Wouldn't `Stream` suffice, given that it is already under `Protocols.Axi4`?